velox/row/tests/UnsafeRowDeserializerTest.cpp

```diff
@@ -918,7 +920,9 @@ class UnsafeRowComplexDeserializerTests : public exec::test::OperatorTestBase {
         batchSize,
         [](vector_size_t row) { return row % 5; },
         [](vector_size_t row, vector_size_t index) {
-          return StringView("str" + std::to_string(row + index));
+          auto str = "string" + std::to_string(row + index);
```

Comment:
check comment below; we should probably wrap this common pattern under a simple API